WeakAuras/RegionTypes/Icon.lua

```diff
@@ -293,8 +292,8 @@ local function modify(parent, region, data)
   end
 
   function region:UpdateSize()
-    local width = region.width * math.abs(region.scalex);
-    local height = region.height * math.abs(region.scaley);
+    local width = region.width * region.scalex;
```

Comment:
The math.abs is there for a "reason". I think the flip animation uses negative scales. (Yes it's rather insane, can't change it.)